CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.5)
```

Comment:
Is this necessary?  That's a pretty new cmake.  What's the minimum version we actually need?

Reply:
turns out 3.5 is not all that new...